### PR TITLE
Update adms request

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Allows the ADMS to have harder but rewarding settings for when people want to have tougher fights
Underground mobs can now spawn from the ADMS when in underground

### CODE REVIEW

**Comment by AMcilraith** [comment]
Requested by me!

File: code/modules/maint_expeditions/adms.dm [hunk 1]
```diff
@@ -6,7 +6,9 @@
 
 /obj/machinery/exploration/adms
 	name = "\improper Anomalous Data Measurement System"
-	desc = "A large piece of equipment for gathering data from anomalous regions. It has an habit of angering the local population pest."
+	desc = "A large piece of equipment for gathering data from anomalous regions. \
+	It has an habit of angering the local population pest. \
```
Comment (Smokingsquirrel) [inline code/modules/maint_expeditions/adms.dm:10]:
```suggestion
	It has a habit of angering the local pest population. \
```

File: code/modules/maint_expeditions/adms.dm [hunk 2]
```diff
@@ -6,7 +6,9 @@
...
+	Has a slider that sets from 0 to 5, depending on the setting more habitents angered but more points are generated."
```
Comment (Smokingsquirrel) [inline code/modules/maint_expeditions/adms.dm:11]:
```suggestion
	Has a slider that sets from 0 to 5, depending on the setting more inhabitants are angered and thus more points generated."
```

**Comment by du-top** [comment]
Could you state a reason for closing?

**Pull request closed without merging**